test/unit/gen/certificaterequest.go

```diff
@@ -139,3 +139,9 @@ func SetCertificateRequestFailureTime(p metav1.Time) CertificateRequestModifier
 		cr.Status.FailureTime = &p
 	}
 }
+
+func SetAnnotations(annotations map[string]string) CertificateRequestModifier {
```

Comment:
I think this may need to be `SetCertificateRequestAnnotations` as it returns a `CertificateRequestModifier` 😬

Reply:
that one exists... how did I look over that :woman_facepalming: